docs/addons/install-addons.md

```diff
@@ -2,13 +2,37 @@
 title: 'Install addons'
 ---
 
-Storybook has [hundreds of reusable addons](https://storybook.js.org/addons) that are packaged as NPM modules. Let's walk through how to extend Storybook by installing and registering addons.
+Storybook has [hundreds of reusable addons](https://storybook.js.org/integrations) packaged as NPM modules. Let's walk through how to extend Storybook by installing and registering addons.
 
-### Using addons
+## Automatic installation
 
-With the exception of preset addons, all addons have the same installation process: install and register.
+Storybook includes a [`storybook add`](../api/cli-options.md#add) command to automate the setup of addons. Several community-led addons can be added using this command, except for preset addons. We encourage you to read the addon's documentation to learn more about its installation process.
```

Comment:
Why can’t it handle preset style addons?

Reply:
If there are any addons where `add` fails, we should figure out what’s going on and fix the issue

Reply:
> Why can’t it handle preset style addons?

I was just going on what was documented here. If that limitation isn't factual, that's great!

Reply:
So that we're all on the same page here, this isn't on us. Our presets are working; what I've encountered during my rounds collecting community addons is basically this, I've run into situations where preset addons failed to register. I cannot know for sure what's causing it, could be an outdated version or something of that nature. Hence why I left it as such so that we don't run into any situations where a user faces a similar question, reach out to us raising an issue/ discussion when the situation is out of our control.

Reply:
Could you please share a scenario where you had this failure? Thanks!